R/opts.R

```diff
@@ -339,19 +340,18 @@ trunc_opts <- function(dist = Fixed(0), default_cdf_cutoff = 0.001,
 #' rt_opts()
 #'
 #' # add a custom length scale
-#' rt_opts(prior = list(mean = 2, sd = 1))
+#' rt_opts(prior = Normal(mean = 2, sd = 1))
 #'
 #' # add a weekly random walk
 #' rt_opts(rw = 7)
-rt_opts <- function(prior = list(mean = 1, sd = 1),
+rt_opts <- function(prior = LogNormal(mean = 1, sd = 1),
```

Comment:
Flagging for me to circle back to - should this really be lognormal now

Reply:
it looks like this is the case in all vignettes. I think probably this should still be Normal